libraries/lib-datahandler/src/main/thrift/sw360.thrift

```diff
@@ -27,6 +27,7 @@ enum RequestStatus {
     FAILED_SANITY_CHECK = 4,
     DUPLICATE = 5,
     DUPLICATE_ATTACHMENT = 6,
+    UPDATE_CLOSED_PROJECT_NOT_ALLOWED = 7,
```

Comment:
This is a project specific message. I would prefer a generic variant, e.g. `CLOSED_UPDATE_NOT_ALLOWED`.

Reply:
Good idea. Now I have made the status more generic but the error message is kept as it is. If there is any change in logic in future then the error message can be modified accordingly.